src/main/webapp/css/style.css

```diff
@@ -28,3 +40,282 @@
 .btn-group {
     margin-top: auto;
 }
+
+
+ .card-header {
+    font-size:30px;
+    white-space: nowrap;
+    overflow: hidden;
+}
+
+.btn-colour-1 {
+  color: #fff;
+  background-color: #2C384A;
+  border-color: #2C384A;
+  font-weight: bold;
+  letter-spacing: 0.05em;
+  border-radius: 0;
+}
+
+/* Wizard Section */
```

Comment:
These seem like Timothy's code. Shouldnt be part of your PR.

Reply:
I think it might be because I made some changes to indentation throughout the whole file